GroupIS1024/Pages/Index.cshtml

```diff
@@ -3,12 +3,14 @@
 @{
     ViewData["Title"] = "FantasyFootball";
 }
-<html>
+<!DOCTYPE html>
+<html lang="en-us">
 <head>
 
     <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```

Comment:
Added Meta Tag for setting the viewport to follow HTML best practice

Source: https://www.w3schools.com/html/html5_syntax.asp